### DISCUSSION TITLE
Local Recording

### DISCUSSION THREAD

**Discussion by oksuzgil** [mynaparrot/plugNmeet-server#284] [General]
"recording_features": {
        "is_allow": false,
        "is_allow_cloud": true,
        "is_allow_local": false,
        "enable_auto_cloud_recording": true
      }
      
      While creating the room, local registration is not displayed when the local registration feature is disabled and the admin user presses the registration button. but if you don't choose cloud and click the start recording button, it records locally.

**Comment by oksuzgil** [comment]
Video Recording Error

Local recording starts when the Start Recording button is clicked and the start recording is clicked without selecting the option.

In the same way, if the option is not selected when the local recording closed room is created while creating the room via API, the local recording is started.

I don't want local recording. That's why I turn it off when creating a room with the api. However, if the user says start recording and clicks start recording again without selecting cloud recording, local recording starts.

If the option is not selected on the recording selection screen, the start recording button should be inactive.

@jibon57

**Reply by jibon57** [reply]
Thanks, have fixed here: https://github.com/mynaparrot/plugNmeet-client/pull/447/commits/ba0367db2619a4feb6b1a17b55a7d9de65275099
